sass/components/_venue_info.scss

```diff
@@ -0,0 +1,40 @@
+.venue {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: $color-bg-primary;
+  height: 650px;
+
+  &__info_title {
+    @media (max-width: 768px) {
+      margin-top: 10px;
+    }
+  }
+
+  button {
+    width: 172px;
+    height: 71px;
+  }
+
+  &__info__description {
+    width: 600px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    text-align: center;
+
+    @media (max-width: 768px) {
+      width: 100%;
+      padding: 20px;
+    }
+  }
+
+  &__tree {
+    position: absolute;
+    top: -0;
+    right: -200px;
+
```

Comment:
could add z-index: -1 or 1 can't remember which one it is that makes it go behind the other, ut it should be the one that makes it go behind the text and such.